components/Player/PlayerTabs.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import TabBar from '../TabBar';
 import { playerPages } from '../Header/Pages';
 
-const playerPagesMapped = (accountId) => playerPages.map((e) => Object.assign({}, e, {
-  route: `/players/${accountId}/${e.name.toLowerCase()}`,
-  label: e.name,
+const playerPagesMapped = (accountId) => playerPages.map(page => ({
+  route: `/players/${accountId}/${page.name.toLowerCase()}`,
```

Comment:
idk what to do with these errors :laughing: 
![image](https://cloud.githubusercontent.com/assets/20866892/18689706/34e50cd4-7f93-11e6-8794-7f44588af920.png)
may I just disable eslint here? hehe